apps/meteor/app/livechat/client/lib/chartHandler.ts

```diff
@@ -108,7 +109,7 @@ const doughnutChartConfiguration = (
 
 type ChartDataSet = {
 	label: string;
-	data: number;
+	data: number[];
```

Comment:
Is this type change expected?

Reply:
Yeah, it was wrong before, but since the files that consumed this type were all in js, no error was raised